public/app/percona/dbaas/components/DBCluster/DBCluster.tsx

```diff
@@ -37,12 +47,13 @@ import { logger } from '@sentry/utils';
 
 export const DBCluster: FC = () => {
   const styles = useStyles(getStyles);
-  const [addModalVisible, setAddModalVisible] = useState(false);
   const [deleteModalVisible, setDeleteModalVisible] = useState(false);
   const [editModalVisible, setEditModalVisible] = useState(false);
   const [logsModalVisible, setLogsModalVisible] = useState(false);
   const [updateModalVisible, setUpdateModalVisible] = useState(false);
   const [selectedCluster, setSelectedCluster] = useState<Cluster>();
+  const { selectedKubernetesCluster } = useSelector(getDBaaS);
+  const [addModalVisible, setAddModalVisible] = useState(selectedKubernetesCluster ? true : false);
```

Comment:
```suggestion
  const [addModalVisible, setAddModalVisible] = useState(selectedKubernetesCluster ? true : false);
```
```suggestion
  const [addModalVisible, setAddModalVisible] = useState(!!selectedKubernetesCluster);
```

Reply:
@fabio-silva Yes, if the user has selected a cluster, then we open the model with the default values of this cluster (previously we opened it with the default value of the first cluster from the list)

In the dbclusters component (dbclusters page) I added useEffect with return function where I reset to null (i.e. as soon as the user leaves this page, the value will be reset to null and the first cluster from the list will be shown in the open modal again by default)